shared/lsif/graphql.ts

```diff
@@ -12,7 +12,7 @@ import { LocationConnectionNode, nodeToLocation } from './conversion'
  * relatively low unless it's a VERY popular library and LSIF data is
  * ubiquitous (which is our goal).
  */
-const MAX_REFERENCE_PAGE_REQUESTS = 20
+const MAX_REFERENCE_PAGE_REQUESTS = 5
```

Comment:
No objection, but why this change? Does removal of caching make the previous limit bad?

Reply:
No, this is kind of a drive-by.

Reply:
Actually I'll revert this line.